frontend/app/src/components/form/fields/relationship.field.tsx

```diff
@@ -26,15 +26,19 @@ const getGenericParentRelationship = (kind?: string) => {
 
   const nodes = store.get(schemaState);
   const schemaData = nodes.find((schema) => schema.kind === kind);
-  return schemaData?.relationships?.find((rel) => rel.kind === "Parent");
+  const parentRelationship = schemaData?.relationships?.find((rel) => rel.kind === "Parent");
+
+  return parentRelationship;
 };
 
 const getParentRelationship = (peer?: string) => {
```

Comment:
getGenericParentRelationship and getParentRelationship does exactly the same thing no?